src/Test2.test.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * @jest-environment jsdom
+ */
+import editTask from './modules/editTask.js';
+import taskStatusUpdate from './modules/taskStatusUpdate.js';
+import list from './modules/list.js';
+import clearCompletedTasks from './modules/clearCompletedTasks.js';
+
+describe('Edit task', () => {
+  let tasks = [];
+  const newTask = {
+    description: 'New task',
+    completed: false,
+    id: tasks.length + 1,
+  };
+
+  test('Clear completed test', () => {
+    document.body.innerHTML = `
+            <ul class="to-do-list">
+            <li id="to-do-title" class="list-item">Today's To Do</li>
+            <li class="list-item">
+                <input class="input" type="text" placeholder="Add to your list...">
+            </li>
+            </ul>
+        <a class="clear">Clear all completed</a>
+        `;
+
+    // pushing new tasks into the array
+    let i = 0;
+    while (i < 8) {
+      const newTask = { description: `Task ${i}`, completed: i % 2 !== 0 };
+      tasks.push(newTask);
+      i += 1;
+    }
+
+    const container = document.querySelector('.to-do-list');
+    tasks.forEach((task) => {
+      list(container, task, '#');
+    });
+
+    // Clearing the completed tasks
+    tasks = clearCompletedTasks(tasks);
+    while (container.children[2]) {
+      container.removeChild(container.children[2]);
+    }
+    tasks.forEach((task) => {
+      list(container, task, '#');
+    });
+    const taskItems = document.querySelectorAll('.description');
+
+    // The test where there were 8 tasks, then deleted 4 completed tasks
+    expect(taskItems).toHaveLength(4);
+  });
+
+  test('Toggle completed status', () => {
+    newTask.completed = false;
+    tasks.push(newTask);
+    taskStatusUpdate(tasks, 0);
+    expect(tasks[0].completed).toBe(true);
```

Comment:
- [ ] **[OPTIONAL]** Also I see that you are testing the status update of the `newTask` to-do, but you have to be careful with the index you choose, as in this case, index 0 doesn't represent what you expect. This is because your `tasks` array is global, so any mutations coming from the test suites will persist between tests, which means that at the beginning of this test, the array has the 4 elements left from the last `clear completed` test. I suggest that you find a way of avoiding this dependency between tests! 🤓 🔧